Examples/Tests/FluxInjection/analysis_flux_injection_3d.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Remi Lehe
+#
+# This file is part of WarpX.
+#
+# License: BSD-3-Clause-LBNL
+
+"""
+This script tests the Gaussian-flux injection (and in particular
+the rejection method in WarpX that we use to generate the right
+velocity distribution).
+
+Two population of particles are injected, with a slightly different
+ratio of u_m/u_th. (This is in order to test the two different
+rejection methods implemented in WarpX, which depend on the u_m/u_th ratio.)
+
+After the particles are emitted with flux injection, this script produces
+histograms of the velocity distribution and compares it with the expected
+velocity distibution (Gaussian or Gaussian-flux depending on the direction
+of space)
+"""
+import os
+import re
+import sys
+
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy.constants import c, m_e, m_p
+from scipy.special import erf
+import yt
+
+sys.path.insert(1, '../../../../warpx/Regression/Checksum/')
+import checksumAPI
+
+yt.funcs.mylog.setLevel(0)
+
+# Open plotfile specified in command line
+fn = sys.argv[1]
+ds = yt.load( fn )
+ad = ds.all_data()
+t_max = ds.current_time.item()  # time of simulation
+
+# Total number of electrons expected:
+# Simulation parameters determine the total number of particles emitted (Ntot)
+flux = 1. # in m^-2.s^-1, from the input script
```

Comment:
It's not necessary, but this could be generalized by having these parameters obtained from the `ds.parameters` dictionary rather than hardcoded.